app/wrappers/Executors.js

```diff
@@ -90,6 +90,12 @@ class Executors {
     executorsRemoved() {
         return this.executorsData.executorsRemoved || [];
     }
+
+    executorsAliasList() {
```

Comment:
This method is only returning executors who have changed their name by deed poll so can we rename it to reflect that?